nautobot_circuit_maintenance/jobs/site_search.py

```diff
@@ -94,9 +94,6 @@ class FindSitesWithMaintenanceOverlap(Job):
 
     Future iterations may include the ability to search for multiple circuit overlaps that would allow for just a single
     circuit to be available.
-
-    Args:
-        Job (Nautobot Job): Nautobot Job parent class
```

Comment:
FWIW I think this is correct to remove. The class inheritance isn't an `arg` to this class's constructor.